front/app/containers/Admin/projects/project/inputImporter/ReviewSection/index.tsx

```diff
@@ -42,26 +42,20 @@ const ReviewSection = ({
   const { formatMessage } = useIntl();
   const [ideaId, setIdeaId] = useState<string | null>(null);
   const [approvals, setApprovals] = useState({ approved: 0, not_approved: 0 });
-  const [importTriggered, setImportTriggered] = useState<boolean>(false);
 
   const {
     data: ideas,
     refetch: refetchIdeas,
     isLoading: isLoadingIdeas,
   } = useImportedIdeas({ projectId, phaseId });
 
-  const jobsChanged = () => {
```

Comment:
First, I memoized this function https://github.com/CitizenLabDotCo/citizenlab/pull/8338/commits/741711cebe10d57119eb428c4b8c746d580529ee

But then I've decided to make the state of this component a bit simpler.